doc/source/tutorial/stats/continuous_sine.rst

```diff
@@ -0,0 +1,19 @@
+
+.. _continuous-sine:
+
+Sine Distribution
+===================
+
+Polar-angle distribution required for ensuring uniform distribution over a sphere.
```

Comment:
Can you add a reference? Not a must but nice to have

Reply:
There isn't a particularly good one that I can think of, and none of the others in `doc/source/tutorial/stats/` seem to have a reference.

Reply:
OK, i wanted to add this comment to the doc string of the distribution. Can you add the refernece i posted in the thread. It states the moments you use in _stats, so that ist useful to add.